Makefile

```
@@ -249,6 +250,16 @@ helm-uninstall:
 	helm uninstall $(HELM_RELEASE_NAME) --namespace $(HELM_NAMESPACE)
 	@echo "==> Helm chart uninstalled: $(HELM_RELEASE_NAME)"
 
+# Install git hooks (lightweight; also called by make deps)
+.PHONY: hooks
+hooks:
```

checkmake (0.3.2)

[warning] Target body for "hooks" exceeds allowed length of 5 lines (6).

(maxbodylength)